### DISCUSSION TITLE
Add support for listing albums

### DISCUSSION THREAD

**Discussion by Cadair** [Radiokot/photoprism-android-client#20] [Ideas]
Firstly thanks for the awesome app. One thing that I use heavily in Photoprism is the albums feature, being able to quickly list the albums (I managed to search for them and then bookmark a couple) would be amazing!

**Comment by Radiokot** [comment]
Thank you for checking out the app. I'm thinking on adding the album shortcuts to the search configuration screen.

**Comment by Radiokot** [comment]
Hey Cadair, please check out the latest release – it has filtering by albums and folders.

**Reply by dfoxg** [reply]
Hi @Radiokot, it would be great if there were an album overview, just like in the PWA of PhotoPrism. Great App by the way!

**Reply by dfoxg** [reply]
Hi @Radiokot should i open a new discussion for a separated album view?

**Reply by Radiokot** [reply]
Hi Daniel. Please, open a new discussion and explain in more details what you mean by album overview.

**Reply by Cadair** [reply]
Thanks for adding the feature! Ideally, I would like folders and albums to be separate, I have a lot of folders and relatively few albums and it's hard to spot the albums for the folders :laughing:

**Reply by Radiokot** [reply]
@Cadair  the app prioritizes favorites and then sorts the items by the last update date. For now, you can make your albums favorite in the web app – they will always be at the beginning of the list.

**Reply by Cadair** [reply]
Thanks that's good to know. I hadn't even realised you could favourite albums!

**Reply by dfoxg** [reply]
I started a new discussion, i hope i could explain my needs: https://github.com/Radiokot/photoprism-android-client/discussions/69

**Comment by ant0nwax** [comment]
hI ,am new to this repo,

In the search many things work flawlessly, just I would like to type, by hitting my touchscreen keyboard, that the expression "Albums" in search is displaying the "Folders" and not the "Albums".

If you need a screenshot, I prefer to work during listening to human voice from an interactive voice call, that requests me to do a screenshot. Do not hesitate to stalk on me and find my phone number, call me and I will add whatever this repo volunteers need, to share whats the challange.

NamASte

**Reply by Radiokot** [reply]
You're weird 😕
